tests/bumper-spec.js

```diff
@@ -468,6 +469,35 @@ describe('Bumper', function () {
       it('should not reject', function () {
         expect(error).to.equal(null)
       })
+
+      describe('when package-lock exists', function () {
+        let madeLock = false
+        beforeEach(function (done) {
+          if (!fs.existsSync('package-lock.json')) {
+            fs.writeFileSync('package-lock.json', '')
```

Comment:
Rather than actually creating a `package-lock.json` file and then removing it, wouldn't it be simpler to use `sinion` to stub `fs.existsSync`? Then you could easily test the case where it returns `true`, and the case where it returns `false`.

This file already has a `sandbox` you can use to stub it.

Reply:
fair enough, let me give it a try!

Reply:
Let me know if you run into any issues, or need an example of stubbing a library method like that.